src/components/Checkbox/Checkbox.tsx

```diff
@@ -44,7 +44,7 @@ export const Checkbox: React.FC<Props> = forwardRef(
         }
 
         return (
-            <div style={style} {...block({ className, asFlexbox })}>
+            <div style={style} {...block({ className, asFlexbox, ...rest })}>
```

Comment:
there are usage of elem here where you removed the props (looking at the previous PR) but you haven't added it back